lib/rails_semantic_logger/extensions/active_support/logger.rb

```diff
@@ -4,15 +4,22 @@ module ActiveSupport
   # More hacks to try and stop Rails from being it's own worst enemy.
   class Logger
     class << self
-      undef :logger_outputs_to?, :broadcast
+      undef :logger_outputs_to?
+
+      # Prevent broadcasting since SemanticLogger already supports multiple loggers
+      if defined(:broadcast)
```

Comment:
Probably there is a typo, it should be `defined?`
```suggestion
      if defined?(:broadcast)
```

Reply:
I believe what you want to use here is `method_defined?`:

```ruby
class Test
  class << self
    def check_defined
      defined? :non_existent_method
    end

    def check_method_defined
      method_defined? :non_existent_method
    end

    def check_method_defined_true
      method_defined? :to_s
    end
  end
end

Test.check_defined
# => "expression"
Test.check_method_defined
# => false
Test.check_method_defined_true
# => true
```

Reply:
Fixed